frontend/src/service/defenceService.ts

```diff
@@ -50,7 +50,12 @@ async function toggleDefence(
 		},
 		body: JSON.stringify({ defenceId, level }),
 	});
-	return response.status === 200;
+	if (response.status !== 200) return null;
+
+	const { chatInfoMessage } =
+		(await response.json()) as ChatInfoMessageResponse;
+
+	return makeChatMessageFromDTO(chatInfoMessage);
```

Comment:
The lovely thing about returning a ChatMessage from all these service functions is that it'd be really easy to return an "error" chat message to go into the chat panel, instead of `null` to be ignored. If only we have time....